integration/tests/main/02-create-package-repository.spec.js

```diff
@@ -0,0 +1,35 @@
+const { test, expect } = require("@playwright/test");
+const { KubeappsLogin } = require("../utils/kubeapps-login");
+const utils = require("../utils/util-functions");
+
+test("Create a new package repository successfully", async ({ page }) => {
+  test.setTimeout(60000);
+
+  // Log in
+  const k = new KubeappsLogin(page);
+  await k.doLogin("kubeapps-operator@example.com", "password", process.env.ADMIN_TOKEN);
+
+  // Go to repos page
+  await page.goto(utils.getUrl("/#/c/default/ns/kubeapps/config/repos"));
```

Comment:
I have set a fixed `/` to start the login so that we keep things simple and clean.
Playing around with `page.goto` introduces randomness into the mix. For example, did try to set a test with a couple of `page.goto` with different urls: sometimes tests would run fine, and sometimes navigation would be reverted to the  homepage.